docs/sources/next/javascript-api/k6-experimental/browser/mouse/up.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'up([options])'
+description: 'Mouse dispatches a `mouseup` event on the current position of the mouse.'
+---
+
+# up([options])
+
+Dispatches a `mouseup` event on the mouse's current position. Mouse can be moved to a different position using [`mouse.move()`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/mouse/move).
```

Comment:
I'm not sure it makes sense to mention `mouseMove` here. What's the link between the two APIs?

Reply:
Explaned here: https://github.com/grafana/k6-docs/pull/1594#discussion_r1608001200.